### DISCUSSION TITLE
Question about Organs packages

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#944] [Developers]
I know that organ packages are just compressed archives that can be opened with zip/rar tools. They are loaded in GO using file/install package. This is fine but I don’t see anywhere the uncompressed organs so I wonder how they are managed. I mean: I don’t believe they are uncompressed on the fly (in temp dir ?) and then deleted on close…
I post this question to understand if it would be better having only compressed samplesets just changing the file extension. 

note: after some search i found interesting posts by Lars Palo on the GO Tool included in the installation. I'm looking at it; however, at first sight it does not answer a basic Q: most samplesets com in .zip or .rar format. They are recognized by GO as packages but they fall in error because the compression ratio/method is usually not set to 0 (GO returns the error "only stored compression supported").
The GOTool i suppose is able to compress and generate organ packages from folder and subfolders: ok, but this means one has to uncompress samplesets downloaded from internet before submitting them to GO tool...
Question is at this point: what is the reald advantage of using Install Organ Package ? Size is the same... organ are loaded into ram no matter they are in folder structures or in one compressed file...
just to understand better. thanks.

**Comment by vpoguru** [comment]
I had a similar issue with the macOS version of the GrandOrgueTool. In order to get the tool working I had to install the tools, which are also required to build the program (I assume that not all of them are required, but I did not test that):

    brew install gettext jack docbook-xsl wxmac cmake pkg-config fftw wavpack
